tests/std/tests/P2674R1_is_implicit_lifetime/test.compile.pass.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <type_traits>
+
+enum UnscopedEnum { Enumerator };
+enum struct ScopedEnum { Enumerator };
+class IncompleteClass;
+struct TrivialClass {};
+struct AggregateClassWithUserProvidedDestructor {
+    ~AggregateClassWithUserProvidedDestructor() {}
+};
+
+using namespace std;
+
+#ifdef __cpp_lib_is_implicit_lifetime
+// this is a test to test whether
+// is_implicit_lifetime_v produces desired results
+// compiles under std namespace
+
+template <typename T>
+constexpr bool test_implicit_lifetime = is_implicit_lifetime_v<T> && is_implicit_lifetime<T>::value;
+
+// Basics (arrays thereof included in Arrays section)
+static_assert(test_implicit_lifetime<int>);
+static_assert(test_implicit_lifetime<int*>);
+static_assert(test_implicit_lifetime<nullptr_t>);
+static_assert(test_implicit_lifetime<UnscopedEnum>);
+static_assert(test_implicit_lifetime<ScopedEnum>);
+static_assert(!test_implicit_lifetime<void (*)()>);

```

Comment:
This line was itself wrong.
```suggestion
static_assert(test_implicit_lifetime<void (*)()>);
```

Reply:
Bugs should be reported upstream, marked as TRANSITION in our sources, and we should avoid asserting wrong answers (which makes it annoying to upgrade to tools when they start returning the right answers). We mark code as "workaround" or "no workaround" to indicate whether it should be removed or preserved, respectively, when the fix becomes available.

Reply:
LLVM-159100 was reported. But the "bug" wasn't reproducible and Clang already correctly behaved in CI, so I decided to close it as invalid. Maybe I was too rushed and it should be temporarily labeled "incomplete".